gdnative-core/src/core_types/vector2.rs

```diff
@@ -97,13 +103,15 @@ impl Vector2Godot for Vector2 {
         )
     }
 
+    /// Returns a perpendicular vector.
     #[inline]
-    fn tangent(self) -> Self {
+    pub fn tangent(self) -> Self {
         Vector2::new(self.y, -self.x)
     }
 
+    /// Returns `self` moved towards `to` by the distance `delta`, clamped by `to`.
     #[inline]
-    fn move_towards(self, to: Vector2, delta: f32) -> Self {
+    pub fn move_towards(self, to: Vector2, delta: f32) -> Self {
```

Comment:
This method is called [`move_toward()` in Godot](https://docs.godotengine.org/en/stable/classes/class_vector2.html#class-vector2-method-move-toward) (was already wrong before).